pom.xml

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-starter-parent</artifactId>
+        <version>2.5.2</version>
+        <relativePath/> <!-- lookup parent from repository -->
+    </parent>
+    <groupId>aluczak</groupId>
+    <artifactId>reservation-system</artifactId>
+    <version>0.0.1-SNAPSHOT</version>
+    <name>reservation-system</name>
+    <description>Reservation System</description>
+    <properties>
+        <java.version>11</java.version>
```

Comment:
dlaczego zdecydowałaś się na 11 a nie na coś nowszego? (nie mówię że to źle)

Reply:
Java 11 to wersja LTS, może być naturalnym wyborem dla nowych projektów. Inna sprawa, że ten projekt nie będzie długo wspierany.